<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proyecto de Visi√≥n Artificial</string>
  </property>
  <widget class="QFrame" name="imageFrameS">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="imageFrameD">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="captureButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Capture</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>110</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Object</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>190</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Object</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="objectList">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>70</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="maxCount">
    <number>3</number>
   </property>
  </widget>
  <widget class="QPushButton" name="renameButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>150</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Rename Object</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>230</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Image</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="imageFrameS_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>700</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Matches</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>290</y>
     <width>180</width>
     <height>217</height>
    </rect>
   </property>
   <property name="text">
    <string> Showing the original
 image keypoints and
 the matching with
 the second image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="load1">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>280</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load1</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="load2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>320</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load2</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="showMat">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>440</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ShowMatches</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>captureButton</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
